Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
I would like the destination interface to remain as is. There is no reason for `S3CmdDestinaion` to implement `Countable`, `IteratorAggregate`. If you want `RotatingStreamDestination` to implement them, that's fine.